spec/helpers/application_helper_spec.rb

```diff
@@ -7,7 +7,7 @@
     end
 
     it 'should inclide the base title' do
-      expect(full_title('foo')).to match(/^hackEDU/)
+      expect(full_title('foo')).to match(/^Hack Club/)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.